packages/components/hub/HubLanding.tsx

```diff
@@ -37,8 +38,19 @@ export const HubLanding: React.FC = () => {
       <View style={{ flex: 1 }}>
         {!!banner && (
           <>
-            <SpacerColumn size={7} />
-            <HubBanner banner={banner} />
+            <Link to={banner?.url} style={{ width: "100%", maxHeight: 500 }}>
+              <OptimizedImage
+                sourceURI={banner?.image}
+                width={width}
```

Comment:
this value should not be dynamic, the resizing proxy will create and cache an image for every new width passed

Reply:
There is no change on `HubLanding.tsx` on this PR (I rolled back), it's out of scope